packages/grid/src/vaadin-grid-column-resizing-mixin.js

```diff
@@ -57,9 +57,11 @@ export const ColumnResizingMixin = (superClass) =>
         const targetCell = columnRowCells.filter((cell) => cell._column === column)[0];
         // Resize the target column
         if (targetCell.offsetWidth) {
-          const style = window.getComputedStyle(targetCell);
+          // Resize handle uses pseudo-element as its clickable area
+          const handleStyle = getComputedStyle(handle, '::before');
```

Comment:
I wonder if `handle.getBoundingClientRect()` would be simpler, to not assume the use of a pseudo element.

Reply:
> I wonder if `handle.getBoundingClientRect()` would be simpler

It would return `3` as the width of resize handle itself is 3px. That's why we need to check for pseudo element.

Reply:
> I suppose the same would apply for the cell size as well.

Even if we set `width: auto` on `targetCell._content`, the `getBoundingClientRect()` for it would return actual width which would make it impossible to make the column really narrow, especially with a long header.